website/docs/workflows/reference/workflow-triggers.md

```diff
@@ -0,0 +1,82 @@
+---
+description: This page provides detailed information on workflow triggers available in Appsmith.
+title: Workflow Triggers  
+hide_title: true  
+toc_min_heading_level: 2  
+toc_max_heading_level: 3  
+---
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Workflow Triggers</h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+
+Workflow triggers start workflow runs based on specific conditions or events. This page provides information about the workflow triggers available in Appsmith, including their configurations, parameters, and usage examples.
+
+## Webhook trigger
+
+A webhook trigger allows external services to start a workflow by sending an HTTP request to a specific URL. This trigger is useful for integrating with applications and services that can send HTTP requests when certain events occur. For example, you can set up a webhook trigger to automatically start a workflow whenever a user signs up on your website, allowing you to send a welcome email or update a database in real time.
+
+### Enable webhook trigger
+
+Toggle the switch to enable the webhook trigger. Once enabled, Appsmith generates a unique identifier URL for the workflow.
+
+### URL
+
+Copy the provided endpoint URL to configure the webhook in your external service. This unique URL includes an API key for authentication and is not shown again for security reasons, so be sure to save it for future reference. If you forget to note the URL, you can regenerate it and must update all applications or services to use the new URL. To configure the workflow, use the HTTP POST method along with the webhook URL. For more information, see [Create Basic Workflow](/workflows/tutorials/create-workflow) using the webhook trigger.
```

Comment:
Just POST request also works here

Reply:
The API key can also be used as a bearer token, we should mention that as well for users who want to use that.

@nsarupr  - please confirm.

Reply:
I didn't understand 
> Just POST request also works here
I've mentioned `HTTP POST Method` which means POST request only.